api/lib/infrastructure/repositories/jury-certification-repository.js

```diff
@@ -152,3 +163,16 @@ function _toComplementaryCertificationCourseResultForJuryCertification(complemen
     commonComplementaryCertificationCourseResultsForJuryCertification,
   ];
 }
+
+async function _getBadgeKeyAndLabelsGroupedByTargetProfile() {
+  const result = await knex('complementary-certification-badges')
+    .select({
+      aggregate: knex.raw(
+        `json_agg(json_build_object('key', "badges"."key", 'label', "complementary-certification-badges"."label") order by "badges".id)`
+      ),
+    })
+    .join('badges', 'badges.id', 'complementary-certification-badges.badgeId')
+    .groupBy('badges.targetProfileId');
+
+  return result.map(({ aggregate }) => aggregate);
```

Comment:
On peut eviter le map avec le pluck (en mettant le return en haut)
```suggestion
   .groupBy('badges.targetProfileId')
   .pluck('aggregate');
```

Reply:
J'ai pas réussi à utiliser un `pluck` ici avec un select sur un aggregat.